extlib/lwp/product/runtime/eclipse/plugins/com.ibm.xsp.theme.bootstrap/src/com/ibm/xsp/theme/bootstrap/themes/BlankBootstrap3.theme

```diff
@@ -0,0 +1,839 @@
+<!--                                                                     -->
+<!-- Licensed under the Apache License, Version 2.0 (the "License");     -->
+<!-- you may not use this file except in compliance with the License.    -->
+<!-- You may obtain a copy of the License at:                            -->
+<!--                                                                     -->
+<!-- http://www.apache.org/licenses/LICENSE-2.0                          -->
+<!--                                                                     -->
+<!-- Unless required by applicable law or agreed to in writing, software -->
+<!-- distributed under the License is distributed on an "AS IS" BASIS,   -->
+<!-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or     -->
+<!-- implied. See the License for the specific language governing        -->
+<!-- permissions and limitations under the License.                      -->
+<!--                                                                     -->
+<!-- ******************************************************************* -->
+<theme>
+
+	<!-- ====================  BEGIN::RESOURCES ================ -->
+
+    <!-- xpages core styles -->
+    <resource>
+        <content-type>text/css</content-type>
+		<href>/.ibmxspres/.extlib/responsive/xpages/css/xsp-core.css</href>
+    </resource>
+
+    <!-- bootstrap core styles -->
+		<!-- Default Bootstrap css files are not included -->
+		<!-- Supply your own bootstrap css by extending this theme-->
+
+		<content-type>text/css</content-type>
+		<href>/.ibmxspres/.extlib/responsive/bootstrap3/css/bootstrap-theme.min.css</href>
+	</resource>
+
+	<!-- dbootstrap styles -->
+ 	<resource dojoTheme="true">
+		<content-type>text/css</content-type>
+		<href>/.ibmxspres/.extlib/responsive/dijit/dbootstrap-0.1.1/theme/dbootstrap/dbootstrap.css</href>
+	</resource>
+
+	<!-- dbootstrap main js -->
+	<!--
+	<resource dojoTheme="true">
+		<content-type>application/x-javascript</content-type>
+		<href>/.ibmxspres/.extlib/responsive/dijit/dbootstrap-0.1.1/main.js</href>
+	</resource>
+	-->
+
+	<!-- xpages/bootstrap mixin styles -->
+	<resource>
+		<content-type>text/css</content-type>
+		<href>/.ibmxspres/.extlib/responsive/xpages/css/xsp-mixin.css</href>
+	</resource>
+
+	<!-- xpages/bootstrap IE11 styles -->
+	<!-- TODO fix user agent check in 9.0.X to use proper method -->
+	<resource rendered="#{javascript:context.getUserAgent().getUserAgent().contains('Trident');}">
+		<content-type>text/css</content-type>
+		<href>/.ibmxspres/.extlib/responsive/xpages/css/xsp-ie11.css</href>
+	</resource>
+
+	<!-- jquery core js -->
+	<resource>
+		<content-type>application/x-javascript</content-type>
+		<href>/.ibmxspres/.extlib/responsive/jquery/jquery.min.js</href>
+	</resource>
+
+	<!-- bootstrap core js -->
+	<resource>
+		<content-type>application/x-javascript</content-type>
+		<href>/.ibmxspres/.extlib/responsive/bootstrap3/js/bootstrap.min.js</href>
+	</resource>
+
```

Comment:
Would you want to remove the default bootstrap javascript as well, so that you can inject your own custom bootstrap js file here?